src/utils/date.ts

```diff
@@ -1,13 +1,89 @@
-const getDateDisplayValue = (date: string | number) => new Date(date).toISOString().split('T')[0];
+import { DateTime } from 'luxon';
 
-export const getTodaysDateFormatted = () => {
-  return getDateDisplayValue(Date.now());
+/**
+ * Helper util that returns default time zone as a fallback.
+ */
+const tz = (timeZoneId?: string) => timeZoneId || 'Etc/UTC';
```

Comment:
Would it be useful for this function to validate that `timeZoneId` is a valid identifier?

Reply:
I thought about it, the concept of valid time zones is now within terraware-web (returned from BE). we can iterate on this for sure.